1_crosswalk_fetch/src/fetch_crosswalk.R

```diff
@@ -214,12 +214,27 @@ fetch_wqp_lake_sites <- function(ind_file, characteristicName, bBox, dummy){
   message('warning, avoiding geojson output due to issue with services 2020-01-19
           hacking resultCount as 200')
   # when switching back in the future, seems lat and lon were the names for LatitudeMeasure & LongitudeMeasure
-  lake_sites_sf <- whatWQPsites(siteType = "Lake, Reservoir, Impoundment", characteristicName = characteristicName,
-                                bBox = bBox) %>%
+
+  # this is breaking with a single bbox query when adding DRB...uggg
+  bbox1 <- bBox
+  bbox1[3] <- bbox1[1] + diff(bBox[c(1,3)])/2
+  bbox2 <- bBox
+  bbox2[1] <- bbox1[3]
```

Comment:
This seems like a reasonable start on a general fix for too-big bboxes: sure, split it in half! And I think we can make it more flexible in the future as we add more locations.

Reply:
How long do these WQP calls take? If it's a long time, you might consider a custom split to make a small bbox around the DRB so we're not pulling all the data from the Northeast. We'll have to revise this code again for more generality if and when the domain grows again, anyway, so I don't see any harm in making it fairly specific to today's challenge for now.

Reply:
But also, if it's already pulled, let sleeping dogs lie.